analyzers/tests/SonarAnalyzer.UnitTest/Rules/AzureFunctionsLogFailuresTest.cs

```diff
@@ -25,10 +25,61 @@
 
 namespace SonarAnalyzer.UnitTest.Rules
 {
-    [TestClass]
-    public class AzureFunctionsLogFailuresTest
+    public class FrameworkTestClassAttribute : TestClassAttribute
```

Comment:
This is an unrelated idea, how to get rid of the `#if NET' conditions in the test. If this is considered useful it should go into a separate PR.

Reply:
As discussed offline, I have more lightweight idea how to do this later.